examples/burial-form/ReviewPage.jsx

```diff
@@ -440,10 +439,37 @@ export default function ReviewPage(props) {
               <Link to={page.pageUrl+'?edit=true&source='+page.id} className='vads-u-margin-bottom--1p5 review-page--page-heading--link'>Edit</Link>
             </div>
 
-            {bufferFields(page.fields)}
-          </section>
+                {bufferFields(page.fields)}
+              </section>
+            )
+          }
         )}
-        ) }
+
+        <div>
+          <div className="vads-u-padding-y--2">
```

Comment:
It might be more accessible to replace this `div` with a `p`, since there's a sentence here.